src/components/Settings/SettingsMain.tsx

```diff
@@ -50,6 +52,9 @@ const SettingsMain: React.FC = () => {
   const { data, error, revalidate } = useSWR<MainSettings>(
     '/api/v1/settings/main'
   );
+  const { data: regions, error: regionsError } = useSWR<Region[]>(
+    '/api/v1/discover/regions'
+  );
```

Comment:
I felt like it didn't make sense for the main settings endpoint to return the regions so I added a second API call here. Please let me know if there is a way to improve this.